samwise.js

```diff
@@ -1,5 +1,6 @@
-const oddNumbers = numbers.filter(num => num % 2 !== 0);
```

Comment:
SCRIPT-This function might benefit from using async/await instead of callbacks for better readability.